ophyd/signal.py

```diff
@@ -64,7 +64,7 @@ def _repr_info(self):
         yield ('value', self.value)
         yield ('timestamp', self.timestamp)
 
-    def get(self):
+    def get(self, **kwargs):
```

Comment:
Ah, thanks for the clarification.  I might suggest that future you would probably want a note in here why **kwargs is in this function